src/providers/data_provider_be.c

```diff
@@ -472,6 +472,79 @@ static void signal_be_reset_offline(struct tevent_context *ev,
     check_if_online(ctx, 0);
 }
 
+static void watch_update_resolv(const char *filename, void *arg)
```

Comment:
Was this block of code moved intentionally? Diff is difficult to read...

Reply:
Yes, `watch_config_files` which calls it is needed by `be_process_init`.

Previous call order was be_process_init -> dp_initialized -> watch_config_files and file order be_process_init | ... watch_config_files | dp_initialized. With dp_initialized gone, I reordered it.